deploy/eks/simple-cluster.yml

```diff
@@ -5,10 +5,11 @@ kind: ClusterConfig
 metadata:
   name:
   region: eu-west-1
+  version: "1.23"
```

Comment:
this is the version we support, required to ensure actual findings matches our current expected findings